app/client/src/routes/home.tsx

```diff
@@ -1,16 +1,511 @@
+import { useCallback, useEffect, useReducer, useRef, useState } from 'react';
+import Select from 'react-select';
 // components
-import { Loading } from "components/loading";
+import Alert from 'components/alert';
+import Button from 'components/button';
+import Checkbox from 'components/checkbox';
+import CopyBox from 'components/copyBox';
+import InfoTooltip from 'components/infoTooltip';
+import { Loading } from 'components/loading';
+import RadioButtons from 'components/radioButtons';
+import RangeSlider from 'components/rangeSlider';
+import Summary from 'components/summary';
 // contexts
-import { useContentState } from "contexts/content";
+import { useContentState } from 'contexts/content';
+// types
+import type { Dispatch, ReactNode } from 'react';
+// config
+import { options as listOptions, profiles } from 'config';
 
+/*
+## Types
+*/
+interface InputState {
+  fileFormat: Option<ReactNode, string> | null;
+  dataProfile: Option<JSX.Element, keyof typeof profiles> | null;
+  state: Readonly<Option<string, string>[]> | null;
+}
+
+type InputAction =
+  | { type: 'fileFormat'; payload: Option<ReactNode, string> }
+  | {
+      type: 'dataProfile';
+      payload: Option<JSX.Element, keyof typeof profiles> | null;
+    }
+  | { type: 'initialize'; payload: InputState }
+  | { type: 'reset' }
+  | { type: 'state'; payload: Readonly<Option<string, string>[]> };
+
+type URLQueryParam = [string, URLQueryArg];
+
+interface URLQueryState {
+  [field: string]: URLQueryArg | URLQueryArg[];
+}
+
+type URLQueryArg = string | number | boolean;
+
+type InputValue = URLQueryArg | URLQueryArg[] | null;
+
+/*
+## Constants
+*/
+const defaultFileFormat = 'csv';
+
+const controlFields = ['dataProfile', 'fileFormat'];
+
+/*
+## Utilities
+*/
+// Converts a JSON object into a parameter string
+function buildQueryString(query: URLQueryState, includeControl = true) {
+  const paramsList: Array<URLQueryParam> = [];
+  Object.entries(query).forEach(([field, value]) => {
+    if (!includeControl && controlFields.includes(field)) return;
```

Comment:
```suggestion
    if (!includeControl && field === 'dataProfile') return;
```

I think we want to include the `fileFormat` parameter in the API URL. 

There may be a better way to accomplish this, but the above suggestion will work.